src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs

```diff
@@ -39,12 +39,13 @@ ICatalogService catalogService
         /// <remarks>Get common and virtual Catalogs list with minimal information included. Returns array of Catalog</remarks>
 		[HttpGet]
         [Route("")]
-        public async Task<ActionResult<Catalog[]>> GetCatalogs(string sort = null, int skip = 0, int take = 20)
+        public async Task<ActionResult<Catalog[]>> GetCatalogs(string sort = null, int skip = 0, int take = 1000)
```

Comment:
Shouldn't we change that condition on the client side?
I think it is safer to explicitly specify this value on the client if we need more.

Reply:
reverted